recipes/recipes_emscripten/lfortran/recipe.yaml

```diff
@@ -0,0 +1,64 @@
+context:
+  name: lfortran
+  version: "0.34.0"
+
+package:
+  name: "{{ name }}"
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/lfortran/lfortran/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d62ddefc3a5e62babdc753400d6733292971db00a7f15456fa74445d8e195433
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
+    - cmake
+    - make   # [unix]
+    - ninja
+    - bison=3.4
+    - re2c
+    - python
+    - toml
+    - zlib=1.2.13
+    - lfortran=0.34.0
```

Comment:
You can use the `{{ version }}` here directly for `lfortran`. This will ensure that the `.mod` files used are compatible with the wasm build.